app/controllers/registration_controller.rb

```diff
@@ -7,18 +7,23 @@ def new
   end
 
   def create
-    @user = User.new(params[:user])
-    if @user.save
-      flash[:notice] = I18n.t('flash.user.create.success')
-      sign_in(@user)
-      if @user.is_business?
-        redirect_to new_user_business_path(@user)
+    if verify_recaptcha
+      @user = User.new(params[:user])
+      if @user.save
+        flash[:notice] = I18n.t('flash.user.create.success')
+        sign_in(@user)
+        if @user.is_business?
```

Comment:
Three levels of conditionals is considered a code smell. Possible opportunity for some extraction of logic here.